outlookmail/utils.bal

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/lang.'int;
+import ballerina/io;
+
+isolated function sendRequestGET(http:Client httpClient, string resources) returns @tainted json|error {
+    return httpClient->get(resources, targetType = json);
+}
+
+isolated function getOutlookClient(Configuration config, string baseUrl) returns http:Client|error {
+    http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig clientConfig = config.clientConfig;
+    http:ClientSecureSocket? socketConfig = config?.secureSocketConfig;
+    return check new (baseUrl, {
+        auth: clientConfig,
+        secureSocket: socketConfig
+    });
+}
+
+isolated function getRecipientListAsRecord(string comment, string[] addressList) returns ForwardParamsList {
+    Recipient[] recipients = [];
+    foreach string address in addressList {
+        EmailAddress emailAddress = {
+            address: address
+        };
+        Recipient recipient = {
+            emailAddress: emailAddress
+        };
+        recipients.push(recipient);
+    }
+    return {comment: comment, toRecipients: recipients};
+}
+
+isolated function uploadByteArray(byte[] file, UploadSession session) returns @tainted error? {
+    int currentPosition = 0;
+    int size = 3000000;
```

Comment:
Seems `3000000` is used in many places.
Please create a constant for this and use that

Reply:
Added to constants.bal